Makefile

```diff
@@ -1,10 +1,23 @@
 .DEFAULT_GOAL := all
 
+GOKS_KONG_GATEWAY_VERSION ?= 3.0.0-ubuntu
 
 .PHONY: gen-lua-tree
 gen-lua-tree:
-	docker build -t goks -f build/Dockerfile .
-	docker run -it -v $(PWD):/goks goks
+	@cd docker && \
+		docker build \
+			--build-arg GOKS_KONG_GATEWAY_VERSION=${GOKS_KONG_GATEWAY_VERSION} \
+			-t kong-gateway-oss-source \
+			-f Dockerfile.kong-gateway-oss-source \
+			.
+	@docker run \
+		-it \
```

Comment:
aren't these two flags ('i' and 't') useful only for interactivity?  they feel a bit weird on a script.

Reply:
Good call, was cleaning up and left that in.

Reply:
Squashed within original commit